src/components/TxRow.js

```diff
@@ -30,7 +30,7 @@ class TxRow extends React.Component {
     return (
       <tr onClick={(e) => this.handleClickTr(this.props.tx.tx_id)}>
         <td className="pr-3">{this.props.tx.tx_id}</td>
-        <td className="pr-3">{dateFormatter.parseTimestamp(this.props.tx.timestamp)}</td>
+        <td className="pr-3">{hathorLib.dateFormatter.parseTimestamp(this.props.tx.timestamp)}</td>
```

Comment:
I don't feel that `dateFormatter` should be in hathorLib.

Reply:
The same as the comment above, I agree it's not a lib method but we use in both wallets, so I really don't know what's best.

Reply:
Ok. Make sense.